### PR TITLE
Add return types to bridges

### PR DESCRIPTION
**Pull request by nicolas-grekas**
| Q             | A
| ------------- | ---
| Branch?       | 6.0
| Bug fix?      | no
| New feature?  | yes
| Deprecations? | no
| Tickets       | #40154
| License       | MIT
| Doc PR        | -

Extracted from #42496

### CODE REVIEW

**Comment by nicolas-grekas** [comment]
:green_circle:

File: src/Symfony/Bridge/Twig/NodeVisitor/Scope.php [hunk 1]
```diff
@@ -40,7 +40,7 @@ public function enter()
      *
      * @return self|null
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:41]:
```suggestion
```

File: src/Symfony/Bridge/Twig/NodeVisitor/Scope.php [hunk 2]
```diff
@@ -70,7 +70,7 @@ public function set(string $key, mixed $value)
      *
      * @return bool
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:71]:
```suggestion
```

File: src/Symfony/Bridge/Twig/NodeVisitor/Scope.php [hunk 3]
```diff
@@ -88,7 +88,7 @@ public function has(string $key)
      *
      * @return mixed
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:89]:
```suggestion
```

File: src/Symfony/Bridge/Twig/Translation/TwigExtractor.php [hunk 4]
```diff
@@ -87,15 +87,15 @@ protected function extractTemplate(string $template, MessageCatalogue $catalogue
     /**
      * @return bool
      */
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/Translation/TwigExtractor.php:89]:
```suggestion
```

File: src/Symfony/Bridge/Twig/UndefinedCallableHandler.php [hunk 5]
```diff
@@ -71,7 +71,7 @@ class UndefinedCallableHandler
     /**
      * @return TwigFilter|false
      */
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/UndefinedCallableHandler.php:73]:
```suggestion
```

File: src/Symfony/Bridge/Twig/UndefinedCallableHandler.php [hunk 6]
```diff
@@ -83,7 +83,7 @@ public static function onUndefinedFilter(string $name)
     /**
      * @return TwigFunction|false
      */
```
Comment (derrabus) [inline src/Symfony/Bridge/Twig/UndefinedCallableHandler.php:85]:
```suggestion
```

File: src/Symfony/Bridge/Twig/NodeVisitor/Scope.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:41]: I'll delegate all similar changes to php-cs-fixer when it can do it.

Comment (derrabus) [reply src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:41]: Okay. It's just that you've already removed the return annotations from other methods in this PR, but not here.

**Review by derrabus — APPROVED** [review]

File: src/Symfony/Bridge/Twig/NodeVisitor/Scope.php (hunk 1 above)
Comment (nicolas-grekas) [reply src/Symfony/Bridge/Twig/NodeVisitor/Scope.php:41]: yep, looks like something to fix for php-cs-fixer when union types are used

**Pull request merged**